usaspending_api/search/delta_models/award_search.py

```diff
@@ -532,6 +534,44 @@
     GROUP BY
         GROUPED_BY_DEFC.award_id
 ) IIJA_DEFC on IIJA_DEFC.award_id = awards.id
+-- Total outlays calculation
+LEFT JOIN (
+    SELECT award_id,
+            COALESCE(total_gross_outlay_amount_by_award_cpe, 0)
+            + COALESCE(total_ussgl487200_down_adj_pri_ppaid_undel_orders_oblig_refund_cpe, 0)
+            + COALESCE(total_ussgl497200_down_adj_pri_paid_deliv_orders_oblig_refund_cpe, 0) AS total_outlays
+        FROM (
+            SELECT faba.award_id,
+                    SUM(
+                        CASE
+                            WHEN sa.is_final_balances_for_fy = TRUE
+                            THEN faba.gross_outlay_amount_by_award_cpe
+                        END
```

Comment:
Any reason not to have this `sa.is_final_balances_for_fy = TRUE` predicate at the subquery level instead of using these `CASE` statements per field?